src/Utils/LocalStorage.js

```diff
@@ -0,0 +1,13 @@
+const LocalStorage = {};
+//the getter method
+LocalStorage.get = key => {
+    return localStorage.getItem(key);
+};
+//the setter method
+LocalStorage.set = (key, value) => {
+    return localStorage.setItem(key, value);
+};
+//the remove method
+LocalStorage.remove = key => {
+    return localStorage.removeItem(key);
+};
```

Comment:
Try making your LocalStorage a class. 
```
export default class LocalStorage {

}
```
which also accepts / returns objects to store in localStorage object

Reply:
I just put a standard localstorage file for now, since i do not exactly know how to use it